ui/raidboss/data/triggers/o9s.js

```diff
@@ -111,6 +111,10 @@
     {
       id: 'O9S Longitudinal Implosion',
       regex: /14:3172:Chaos starts using Longitudinal Implosion/,
+      run: function(data) {
```

Comment:
Just because some people disable triggers, can you put side effect logic in a separate trigger, rather than the one that calls out sides -> front/back?